lib/srv/db/common/auth.go

```diff
@@ -523,14 +566,18 @@ ID as "--db-user", or grant the Teleport Database Service the
 
 If the db user uses passwords, make sure Teleport Database Service has "Cloud
 SQL Admin" GCP IAM role, or "cloudsql.users.update" IAM permission.
-`, sessionCtx.DatabaseUser, err)
+`, databaseUser, err)
 	}
 	return nil
 }
 
 // GetAzureAccessToken generates Azure database access token.
-func (a *dbAuth) GetAzureAccessToken(ctx context.Context, sessionCtx *Session) (string, error) {
-	a.cfg.Log.Debugf("Generating Azure access token for %s.", sessionCtx)
+func (a *dbAuth) GetAzureAccessToken(ctx context.Context, database types.Database, databaseUser string, databaseName string) (string, error) {
```

Comment:
same question as above.

```suggestion
func (a *dbAuth) GetAzureAccessToken(ctx context.Context) (string, error) {
```

And GetElastiCacheRedisToken, GetMemoryDBToken